packages/babel-compiler/.versions

```diff
@@ -0,0 +1,3 @@
+babel-compiler@5.4.7
```

Comment:
Don't think we need to check this file in. I think it's only created when you run `publish` or `publish-for-arch` and not when you publish things as part of a release? Do you remember what created it? It can probably be deleted. (Does this package have binary components?)